README.md

```diff
@@ -72,7 +72,26 @@ See the [CONTRIBUTING] documentation for more information on how to get started.
 Versioning
 ----------
 
-This library follows [Semantic Versioning](http://semver.org/).
+This library follows [Semantic Versioning](http://semver.org/), but with some
+additional qualifications:
+
+1. Components marked with `@BetaApi` are considered to be "0.x" features inside
+   a "1.x" library. This means they can change between minor and patch releases
+   in incompatible ways. These features should not be used by any library "B"
+   that itself has consumers, unless the components of library B that use
+   `@BetaApi` features are also marked with `@BetaApi`. Features marked as
+   `@BetaApi` are on a path to eventually become "1.x" features with the marker
+   removed.
+
+   **Special exception for google-cloud-java**: google-cloud-java is
```

Comment:
Say you have the following:

1. in gax-java, feature A is `@BetaApi`
2. in google-cloud-java, feature B uses feature A from gax-java
3. in other-apis, feature C uses feature A from gax-java
4. in app D, it depends on feature B in google-cloud-java and feature C in other-apis

Now say feature A is renamed.

1. gax-java: feature A is renamed
2. In google-cloud-java, feature B is updated at the same time that the gax-java dependency is updated
3. App D upgrades google-cloud-java to use the latest version (includes updated gax-java)

Now, App D will crash when it uses feature C in other-apis, because it depends on the old version of feature A in gax-java.

This is why you don't want libraries with consumers (the other-apis case) to depend on `@BetaApi` features - other libraries at the same level can depend on other versions of the `@BetaApi` feature (e.g. google-cloud-java), such that there is no version for the final app to pick that can work with everything.